aten/src/ATen/native/README.md

```diff
@@ -58,6 +58,13 @@ signature.
   don't commit them to memory.
 - `Tensor[]`.  A `Tensor[]` argument translates into a C++ argument of type `ArrayRef<Tensor>`
   (a.k.a. `TensorList`)
+- `Tensor` or `Tensor?` must sometimes be annotated to indicate aliasing and mutability.
```

Comment:
there's already a bullet point about Tensor / Tensor? above -- is there a reason this is separate?